packages/next/src/server/lib/router-utils/proxy-request.ts

```diff
@@ -24,7 +24,6 @@ export async function proxyRequest(
     target,
     changeOrigin: true,
     ignorePath: true,
-    xfwd: true,
```

Comment:
`http-proxy` attempts to set these headers, even if they are already present. Since we are setting these headers higher up in `base-server.ts`, this should not be necessary to set here anymore.

See: https://github.com/http-party/node-http-proxy/blob/9b96cd725127a024dabebec6c7ea8c807272223d/lib/http-proxy/passes/web-incoming.js#L58-L86